www/src/pages/docs/2.0/integrations/js.md

```diff
@@ -67,8 +122,33 @@ export default defineConfig({
 
 ### Options
 
-| Name   | Type                | Description                                                                                                          |
-| :----- | :------------------ | :------------------------------------------------------------------------------------------------------------------- |
-| `js`   | `string \| boolean` | Set to a filename, or `false` to disable (default: `index.js`).                                                      |
-| `ts`   | `string \| boolean` | Set to a filename, or `false` to disable (default: `index.d.ts`) _Note: this can’t be enabled if `js` is disabled_). |
-| `json` | `string \| boolean` | Set to a filename, or `false` to disable (default: `false`).                                                         |
+| Name       | Type     | Description                               |
+| :--------- | :------- | :---------------------------------------- |
+| `filename` | `string` | Set to a filename (default: `tokens.js`). |
+
+## Migrating from 0.x
```

Comment:
This is more-or-less a roadmap for now. The JS plugin didn’t get updates in this PR, but “working backwards” and writing out the API I’m shooting for first, really helped clarify some upstream work in the parser to get all this ready.

It’ll be a little bit of a PITA to generate the JS for this, but the design is the hard part, and that’s shaping up nicely.